<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Find</class>
 <widget class="QDialog" name="Find">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>210</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>475</width>
    <height>210</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/SIFTTeal.ico</normaloff>images/SIFTTeal.ico</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>101</width>
     <height>140</height>
    </rect>
   </property>
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_findnext">
      <property name="toolTip">
       <string>Go to the next item found [F3].</string>
      </property>
      <property name="text">
       <string>Find Next</string>
      </property>
      <property name="shortcut">
       <string>F3</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_findprev">
      <property name="toolTip">
       <string>Go to the previous match [F4].</string>
      </property>
      <property name="text">
       <string>Find Prev</string>
      </property>
      <property name="shortcut">
       <string>F4</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_count">
      <property name="toolTip">
       <string>Displays how many items where found.</string>
      </property>
      <property name="text">
       <string>Count</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_findinfiles">
      <property name="toolTip">
       <string>Display .fmls that contain an occurance of the text in a new tab.</string>
      </property>
      <property name="text">
       <string>Find in Files</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_findops">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Find In</string>
   </property>
   <widget class="QRadioButton" name="radioButton_current">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Find the text in the file currently selected and displayed in the main tab of the viewer.</string>
    </property>
    <property name="text">
     <string>Current File</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_selection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>93</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Search the currently selected text in the viewer.</string>
    </property>
    <property name="text">
     <string>Selection</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_startpos">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Start From</string>
   </property>
   <widget class="QRadioButton" name="radioButton_top">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Start search from top of the file or selection.</string>
    </property>
    <property name="text">
     <string>Top</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_bottom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Start search from the bottom of the file or selection.</string>
    </property>
    <property name="text">
     <string>Bottom</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_options">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Search Options</string>
   </property>
   <widget class="QCheckBox" name="checkBox_verbatim">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Search for exactly what is in the Find text box. Matches that are inside a word or otherwise partial finds are ignored.</string>
    </property>
    <property name="text">
     <string>Verbatim</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_case">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Case sensitive search.</string>
    </property>
    <property name="text">
     <string>Case</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_wrap">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wrap</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_re">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RegExp</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_find">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>11</y>
     <width>32</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Find: </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_mode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Search Mode</string>
   </property>
   <widget class="QRadioButton" name="radioButton_normal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Search for a word or phrase.</string>
    </property>
    <property name="text">
     <string>Normal</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_reference">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Search for references of the text. This will display the lines it occurs on in the output box, rather than select each match in the main viewer.</string>
    </property>
    <property name="text">
     <string>References</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox_find">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>271</width>
     <height>29</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="maxVisibleItems">
    <number>10</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtCurrent</enum>
   </property>
   <property name="duplicatesEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_findnext</sender>
   <signal>clicked()</signal>
   <receiver>Find</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_findprev</sender>
   <signal>clicked()</signal>
   <receiver>Find</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_count</sender>
   <signal>clicked()</signal>
   <receiver>Find</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_current</sender>
   <signal>clicked()</signal>
   <receiver>Find</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_selection</sender>
   <signal>clicked()</signal>
   <receiver>Find</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_findinfiles</sender>
   <signal>clicked()</signal>
   <receiver>Find</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
